### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Daniil-A4**
[demo link](https://daniil-a4.github.io/layout_landing-page/)

### CODE REVIEW

**Comment by Daniil-A4** [comment]
@xiimiik tell me please why when page is lauding few first second i see something like this?

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,361 @@
...
+      href="./images/favicon.png"
+    >
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="header__top">
+        <a href="index.html">
```
Comment (denys-danyliuk) [inline src/index.html:24]: use # anchor instead to avoid reload of the page

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,361 @@
...
+            class="header__logo logo"
+          >
+        </a>
+
+        <div class="header__controls">
+          <div class="header__wrapper-phone">
+            <a href="tel:+12345555555"
+              class="header__phone-number">
+              +1 234 555-55-55
+            </a>
```
Comment (denys-danyliuk) [inline src/index.html:36]: https://github.com/mate-academy/layout_landing-page/assets/106310796/18d2f6b0-4999-4d43-bd7f-0f4ae30c0e06

Comment (denys-danyliuk) [reply src/index.html:36]: transition should work both ways

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,361 @@
...
+            <li class="nav__item">
+              <a href="#contacts" class="nav__link">Contacts</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+
+      <div class="menu__bottom">
+        <a href="tel:" class="menu__phone">+1 234 5555-55-55</a>
+        <a href="#" class="menu__text">Book a test ride</a>
```
Comment (denys-danyliuk) [inline src/index.html:99]: this also should start a call

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,361 @@
...
+              <div class="contacts__wrapper-input">
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="contacts__input contacts__input--textarea"
+                  required
+                ></textarea>
+              </div>
+
+              <button type="submit" class="contacts__button button">Send</button>
```
Comment (denys-danyliuk) [inline src/index.html:308]: this button should have greater font-weight, check the design

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,361 @@
...
+      <div class="menu__bottom">
+        <a href="tel:" class="menu__phone">+1 234 5555-55-55</a>
+        <a href="#" class="menu__text">Book a test ride</a>
+      </div>
+    </aside>
+
+    <main class="main">
+      <section class="about-us about-us--retreat" id="about-us">
+        <article class="about-us__container container">
+          <h2 class="about-us__title title">Move Free</h2>
```
Comment (denys-danyliuk) [inline src/index.html:106]: check the design, 320px resolution
Знімок екрана 2023-08-07 о 12 05 24

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,361 @@
...
+                >
+              </div>
+
+              <div class="contacts__wrapper-input">
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="contacts__input contacts__input--textarea"
+                  required
+                ></textarea>
```
Comment (denys-danyliuk) [inline src/index.html:305]: delete outline on this elements
Знімок екрана 2023-08-07 о 12 07 56

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,361 @@
...
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="header__top">
+        <a href="index.html">
+          <img src="./images/logo-white.svg"
+            alt="logo"
+            class="header__logo logo"
+          >
```
Comment (denys-danyliuk) [inline src/index.html:28]:
```suggestion
          <img 
            src="./images/logo-white.svg"
            alt="logo"
            class="header__logo logo"
          >
```

File: src/index.html [hunk 8]
```diff
@@ -1,13 +1,361 @@
...
+        </a>
+
+        <div class="header__controls">
+          <div class="header__wrapper-phone">
+            <a href="tel:+12345555555"
+              class="header__phone-number">
```
Comment (denys-danyliuk) [inline src/index.html:34]:
```suggestion
            <a 
              href="tel:+12345555555"
              class="header__phone-number"
            >
```

File: src/index.html [hunk 9]
```diff
@@ -1,13 +1,361 @@
...
+
+              <p class="compare-bikes__price">$ 2 590</p>
+            </div>
+
+            <div class="compare-bikes__item">
+              <a href="#">
+                <img class="compare-bikes__img"
+                  src="./images/compare-bikes/02.png"
+                  alt="Bike-sporty4"
+                >
```
Comment (denys-danyliuk) [inline src/index.html:140]: fix this indentation as I wrote above in all places

File: src/styles/blocks/header.scss [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+    }
+
+    @include onDesktop() {
+      font-size: 80px;
+      margin-bottom: 20%;
+    }
+  }
+}
+
+.logo {
```
Comment (denys-danyliuk) [inline src/styles/blocks/header.scss:113]: make it as a header element

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

File: src/styles/blocks/header.scss (hunk 10 above)
Comment (Daniil-A4) [reply src/styles/blocks/header.scss:113]: I reuse this styles in few places and just moved this block to global file. Or you are toking about something another?

**Review by xiimiik — CHANGES_REQUESTED** [review]
check font-family for inputs
image
add hover for logo, in menu mode also
image

**Review by loralevitska — APPROVED** [review]
Almost perfect) Check my comments for improvement 

- remove default autocomplete background-color, it breaks design 
image
- remove redundant elements from image (you can download another image)

**Pull request closed without merging**